### ISSUE TITLE
Failed to build iOS app

### ISSUE THREAD

**Issue by yasernet1989** [inspireui/support#4060]
**_Step 1 (require): describe detail issues & screenshots_**
Hi,
Im trying to build the xcode version of my app but im getting this error every time i try to build or archive
The android version of this app is building perfectly.
Other flutter ios apps are building perfectly on same mac. I really dont know whats wrong with this app!!
check out this error and suggest to me whats wrong

""

Building com.hours18.wholesale for device (ios-release)...

Automatically signing iOS for device deployment using specified development team

in Xcode project: SKWH4T63R4

Running pod install...                                             23.9s

Running Xcode build...                                                           └─Compiling, linking and signing...                             

  216.5s (!)                                       

Xcode build done.                                           563.7s

Failed to build iOS app

Error output from Xcode build:

↳

    ** BUILD FAILED **

Xcode's output:

↳

    In file included from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/uni_li

    nks-0.2.0/ios/Classes/UniLinksPlugin.m:1:

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/uni_li

    nks-0.2.0/ios/Classes/UniLinksPlugin.h:4:4: warning: pointer is missing a

    nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified)

    [-Wnullability-completeness]

    + (instancetype)sharedInstance;

       ^

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/uni_li

    nks-0.2.0/ios/Classes/UniLinksPlugin.h:4:4: note: insert '_Nullable' if the

    pointer may be null

    + (instancetype)sharedInstance;

       ^

                    _Nullable 

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/uni_li

    nks-0.2.0/ios/Classes/UniLinksPlugin.h:4:4: note: insert '_Nonnull' if the

    pointer should never be null

    + (instancetype)sharedInstance;

       ^

                    _Nonnull 

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/uni_li

    nks-0.2.0/ios/Classes/UniLinksPlugin.h:7:33: warning: block pointer is

    missing a nullability type specifier (_Nonnull, _Nullable, or

    _Null_unspecified) [-Wnullability-completeness]

          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;

                                    ^

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/uni_li

    nks-0.2.0/ios/Classes/UniLinksPlugin.h:7:33: note: insert '_Nullable' if the

    block pointer may be null

          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;

                                    ^

                                     _Nullable

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/uni_li

    nks-0.2.0/ios/Classes/UniLinksPlugin.h:7:33: note: insert '_Nonnull' if the

    block pointer should never be null

          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;

                                    ^

                                     _Nonnull

    2 warnings generated.

    In file included from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/uni_li

    nks-0.2.0/ios/Classes/UniLinksPlugin.m:1:

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/uni_li

    nks-0.2.0/ios/Classes/UniLinksPlugin.h:4:4: warning: pointer is missing a

    nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified)

    [-Wnullability-completeness]

    + (instancetype)sharedInstance;

       ^

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/uni_li

    nks-0.2.0/ios/Classes/UniLinksPlugin.h:4:4: note: insert '_Nullable' if the

    pointer may be null

    + (instancetype)sharedInstance;

       ^

                    _Nullable 

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/uni_li

    nks-0.2.0/ios/Classes/UniLinksPlugin.h:4:4: note: insert '_Nonnull' if the

    pointer should never be null

    + (instancetype)sharedInstance;

       ^

                    _Nonnull 

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/uni_li

    nks-0.2.0/ios/Classes/UniLinksPlugin.h:7:33: warning: block pointer is

    missing a nullability type specifier (_Nonnull, _Nullable, or

    _Null_unspecified) [-Wnullability-completeness]

          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;

                                    ^

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/uni_li

    nks-0.2.0/ios/Classes/UniLinksPlugin.h:7:33: note: insert '_Nullable' if the

    block pointer may be null

          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;

                                    ^

                                     _Nullable

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/uni_li

    nks-0.2.0/ios/Classes/UniLinksPlugin.h:7:33: note: insert '_Nonnull' if the

    block pointer should never be null

          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;

                                    ^

                                     _Nonnull

    2 warnings generated.

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/rate_m

    y_app-0.6.0+3/ios/Classes/RateMyAppPlugin.m:15:30: warning: 'requestReview'

    is only available on iOS 10.3 or newer [-Wunguarded-availability]

        [SKStoreReviewController requestReview];

                                 ^~~~~~~~~~~~~~

    In module 'StoreKit' imported from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/rate_m

    y_app-0.6.0+3/ios/Classes/RateMyAppPlugin.m:2:

    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Devel

    oper/SDKs/iPhoneOS13.5.sdk/System/Library/Frameworks/StoreKit.framework/Head

    ers/SKStoreReviewController.h:22:1: note: 'requestReview' has been marked as

    being introduced in iOS 10.3 here, but the deployment target is iOS 8.0.0

    + (void)requestReview API_AVAILABLE(ios(10.3), macos(10.14))

    API_UNAVAILABLE(watchos) __TVOS_PROHIBITED;

    ^

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/rate_m

    y_app-0.6.0+3/ios/Classes/RateMyAppPlugin.m:15:30: note: enclose

    'requestReview' in an @available check to silence this warning

        [SKStoreReviewController requestReview];

                                 ^~~~~~~~~~~~~~

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/rate_m

    y_app-0.6.0+3/ios/Classes/RateMyAppPlugin.m:15:6: warning:

    'SKStoreReviewController' is only available on iOS 10.3 or newer

    [-Wunguarded-availability]

        [SKStoreReviewController requestReview];

         ^~~~~~~~~~~~~~~~~~~~~~~

    In module 'StoreKit' imported from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/rate_m

    y_app-0.6.0+3/ios/Classes/RateMyAppPlugin.m:2:

    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Devel

    oper/SDKs/iPhoneOS13.5.sdk/System/Library/Frameworks/StoreKit.framework/Head

    ers/SKStoreReviewController.h:14:110: note: 'SKStoreReviewController' has

    been marked as being introduced in iOS 10.3 here, but the deployment target

    is iOS 8.0.0

    SK_EXTERN_CLASS API_AVAILABLE(ios(10.3), macos(10.14))

    API_UNAVAILABLE(watchos) __TVOS_PROHIBITED @interface

    SKStoreReviewController : NSObject

                                                                                                                 ^

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/rate_m

    y_app-0.6.0+3/ios/Classes/RateMyAppPlugin.m:15:6: note: enclose

    'SKStoreReviewController' in an @available check to silence this warning

        [SKStoreReviewController requestReview];

         ^~~~~~~~~~~~~~~~~~~~~~~

    2 warnings generated.

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/rate_m

    y_app-0.6.0+3/ios/Classes/RateMyAppPlugin.m:15:30: warning: 'requestReview'

    is only available on iOS 10.3 or newer [-Wunguarded-availability]

        [SKStoreReviewController requestReview];

                                 ^~~~~~~~~~~~~~

    In module 'StoreKit' imported from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/rate_m

    y_app-0.6.0+3/ios/Classes/RateMyAppPlugin.m:2:

    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Devel

    oper/SDKs/iPhoneOS13.5.sdk/System/Library/Frameworks/StoreKit.framework/Head

    ers/SKStoreReviewController.h:22:1: note: 'requestReview' has been marked as

    being introduced in iOS 10.3 here, but the deployment target is iOS 8.0.0

    + (void)requestReview API_AVAILABLE(ios(10.3), macos(10.14))

    API_UNAVAILABLE(watchos) __TVOS_PROHIBITED;

    ^

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/rate_m

    y_app-0.6.0+3/ios/Classes/RateMyAppPlugin.m:15:30: note: enclose

    'requestReview' in an @available check to silence this warning

        [SKStoreReviewController requestReview];

                                 ^~~~~~~~~~~~~~

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/rate_m

    y_app-0.6.0+3/ios/Classes/RateMyAppPlugin.m:15:6: warning:

    'SKStoreReviewController' is only available on iOS 10.3 or newer

    [-Wunguarded-availability]

        [SKStoreReviewController requestReview];

         ^~~~~~~~~~~~~~~~~~~~~~~

    In module 'StoreKit' imported from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/rate_m

    y_app-0.6.0+3/ios/Classes/RateMyAppPlugin.m:2:

    /Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Devel

    oper/SDKs/iPhoneOS13.5.sdk/System/Library/Frameworks/StoreKit.framework/Head

    ers/SKStoreReviewController.h:14:110: note: 'SKStoreReviewController' has

    been marked as being introduced in iOS 10.3 here, but the deployment target

    is iOS 8.0.0

    SK_EXTERN_CLASS API_AVAILABLE(ios(10.3), macos(10.14))

    API_UNAVAILABLE(watchos) __TVOS_PROHIBITED @interface

    SKStoreReviewController : NSObject

                                                                                                                 ^

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/rate_m

    y_app-0.6.0+3/ios/Classes/RateMyAppPlugin.m:15:6: note: enclose

    'SKStoreReviewController' in an @available check to silence this warning

        [SKStoreReviewController requestReview];

         ^~~~~~~~~~~~~~~~~~~~~~~

    2 warnings generated.

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/flutte

    r_webview_plugin-0.3.11/ios/Classes/FlutterWebviewPlugin.m:92:22: warning:

    incompatible pointer to integer conversion assigning to 'BOOL' (aka 'signed

    char') from 'id _Nullable' [-Wint-conversion]

        _enableAppScheme = call.arguments[@"enableAppScheme"];

                         ^ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/flutte

    r_webview_plugin-0.3.11/ios/Classes/FlutterWebviewPlugin.m:434:98: warning:

    values of type 'NSInteger' should not be used as format arguments; add an

    explicit cast to 'long' instead [-Wformat]

        [channel invokeMethod:@"onHttpError" arguments:@{@"code": [NSString

        stringWithFormat:@"%ld", error.code], @"url": url}];

                                                                                               ~~~

                                                                                               ^~~~~~~~~~

                                                                                               %ld

                                                                                               (long)

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/flutte

    r_webview_plugin-0.3.11/ios/Classes/FlutterWebviewPlugin.m:442:98: warning:

    values of type 'NSInteger' should not be used as format arguments; add an

    explicit cast to 'long' instead [-Wformat]

        [channel invokeMethod:@"onHttpError" arguments:@{@"code": [NSString

        stringWithFormat:@"%ld", error.code], @"error":

        error.localizedDescription}];

                                                                                               ~~~

                                                                                               ^~~~~~~~~~

                                                                                               %ld

                                                                                               (long)

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/flutte

    r_webview_plugin-0.3.11/ios/Classes/FlutterWebviewPlugin.m:450:106: warning:

    values of type 'NSInteger' should not be used as format arguments; add an

    explicit cast to 'long' instead [-Wformat]

                [channel invokeMethod:@"onHttpError" arguments:@{@"code":

                [NSString stringWithFormat:@"%ld", response.statusCode], @"url":

                webView.URL.absoluteString}];

                                                                                                       ~~~

                                                                                                       ^~~~~~~~~~

                                                                                                       ~~~~~~~~~

                                                                                                       %ld

                                                                                                       (long)

    4 warnings generated.

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/flutte

    r_statusbarcolor-0.2.3/ios/Classes/FlutterStatusbarcolorPlugin.m:15:37:

    warning: implicit conversion from 'long long' to 'NSInteger' (aka 'int')

    changes value from 38482458385 to -172247279 [-Wconstant-conversion]

    static NSInteger statusBarViewTag = 38482458385;

                     ~~~~~~~~~~~~~~~~   ^~~~~~~~~~~

    1 warning generated.

    error: the following command failed with exit code 0 but produced no further

    output

    CompileC

    /Users/hinetsmartsolutions/Library/Developer/Xcode/DerivedData/Runner-brolrq

    ntkohexeagyddincnwgplq/Build/Intermediates.noindex/Pods.build/Release-iphone

    os/firebase_auth.build/Objects-normal/armv7/FLTFirebaseAuthPlugin.o

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_auth-0.15.3/ios/Classes/FLTFirebaseAuthPlugin.m normal armv7 objective-c

    com.apple.compilers.llvm.clang.1_0.compiler

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_database-3.1.3/ios/Classes/FLTFirebaseDatabasePlugin.m:13:79: warning:

    values of type 'NSInteger' should not be used as format arguments; add an

    explicit cast to 'long' instead [-Wformat]

      return [FlutterError errorWithCode:[NSString stringWithFormat:@"Error

      %ld", error.code]

                                                                            ~~~

                                                                            ^~~~~~~~~~

                                                                            %ld

                                                                            (long)

    1 warning generated.

    : warning: 'testDevices' is deprecated: Use

    GADMobileAds.sharedInstance.requestConfiguration.testDeviceIdentifiers.

    [-Wdeprecated-declarations]

        request.testDevices = testDevices;

                ^

    In module 'GoogleMobileAds' imported from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:

    /Users/hinetsmartsolutions/whole1/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/

    GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADReques

    t.h:88:71: note: 'testDevices' has been explicitly marked deprecated here

    @property(nonatomic, copy, nullable) NSArray<NSString *> *testDevices

    GAD_DEPRECATED_MSG_ATTRIBUTE(

                                                                          ^

    In module 'GoogleMobileAds' imported from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:

    /Users/hinetsmartsolutions/whole1/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/

    GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMob

    ileAdsDefines.h:25:56: note: expanded from macro

    'GAD_DEPRECATED_MSG_ATTRIBUTE'

    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))

                                                           ^

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.m:91:15: warning: 'birthday'

    is deprecated [-Wdeprecated-declarations]

          request.birthday =

                  ^

    In module 'GoogleMobileAds' imported from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:

    /Users/hinetsmartsolutions/whole1/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/

    GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADReques

    t.h:95:55: note: 'birthday' has been explicitly marked deprecated here

    @property(nonatomic, copy, nullable) NSDate *birthday

    GAD_DEPRECATED_ATTRIBUTE;

                                                          ^

    In module 'GoogleMobileAds' imported from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:

    /Users/hinetsmartsolutions/whole1/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/

    GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMob

    ileAdsDefines.h:33:49: note: expanded from macro 'GAD_DEPRECATED_ATTRIBUTE'

    #define GAD_DEPRECATED_ATTRIBUTE __attribute__((deprecated))

                                                    ^

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.m:106:19: warning: 'gender'

    is deprecated [-Wdeprecated-declarations]

              request.gender = genderValue;

                      ^

    In module 'GoogleMobileAds' imported from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:

    /Users/hinetsmartsolutions/whole1/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/

    GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADReques

    t.h:92:47: note: 'gender' has been explicitly marked deprecated here

    @property(nonatomic, assign) GADGender gender GAD_DEPRECATED_ATTRIBUTE;

                                                  ^

    In module 'GoogleMobileAds' imported from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:

    /Users/hinetsmartsolutions/whole1/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/

    GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMob

    ileAdsDefines.h:33:49: note: expanded from macro 'GAD_DEPRECATED_ATTRIBUTE'

    #define GAD_DEPRECATED_ATTRIBUTE __attribute__((deprecated))

                                                    ^

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.m:116:14: warning:

    'tagForChildDirectedTreatment:' is deprecated:  use

    [GADMobileAds.sharedInstance.requestConfiguration

    tagForChildDirectedTreatment] [-Wdeprecated-declarations]

        [request tagForChildDirectedTreatment:childDirected.boolValue];

                 ^

    In module 'GoogleMobileAds' imported from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:

    /Users/hinetsmartsolutions/whole1/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/

    GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADReques

    t.h:126:5: note: 'tagForChildDirectedTreatment:' has been explicitly marked

    deprecated here

        GAD_DEPRECATED_MSG_ATTRIBUTE(

        ^

    In module 'GoogleMobileAds' imported from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:

    /Users/hinetsmartsolutions/whole1/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/

    GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMob

    ileAdsDefines.h:25:56: note: expanded from macro

    'GAD_DEPRECATED_MSG_ATTRIBUTE'

    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))

                                                           ^

    4 warnings generated.

    1 warning generated.

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRewardedVideoAdWrapper.m:75:66: warning:

    implicit conversion loses integer precision: 'FLTRewardedVideoAdStatus' to

    'int' [-Wshorten-64-to-32]

          (NSString *)rewardedStatusToString[[NSNumber

          numberWithInt:_rewardedStatus]];

                                             ~

                                             ^~~~~~~~~~~~~~~

    1 warning generated.

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.m:71:13: warning:

    'testDevices' is deprecated: Use

    GADMobileAds.sharedInstance.requestConfiguration.testDeviceIdentifiers.

    [-Wdeprecated-declarations]

        request.testDevices = testDevices;

                ^

    In module 'GoogleMobileAds' imported from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:

    /Users/hinetsmartsolutions/whole1/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/

    GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADReques

    t.h:88:71: note: 'testDevices' has been explicitly marked deprecated here

    @property(nonatomic, copy, nullable) NSArray<NSString *> *testDevices

    GAD_DEPRECATED_MSG_ATTRIBUTE(

                                                                          ^

    In module 'GoogleMobileAds' imported from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:

    /Users/hinetsmartsolutions/whole1/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/

    GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMob

    ileAdsDefines.h:25:56: note: expanded from macro

    'GAD_DEPRECATED_MSG_ATTRIBUTE'

    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))

                                                           ^

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.m:91:15: warning: 'birthday'

    is deprecated [-Wdeprecated-declarations]

          request.birthday =

                  ^

    In module 'GoogleMobileAds' imported from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:

    /Users/hinetsmartsolutions/whole1/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/

    GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADReques

    t.h:95:55: note: 'birthday' has been explicitly marked deprecated here

    @property(nonatomic, copy, nullable) NSDate *birthday

    GAD_DEPRECATED_ATTRIBUTE;

                                                          ^

    In module 'GoogleMobileAds' imported from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:

    /Users/hinetsmartsolutions/whole1/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/

    GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMob

    ileAdsDefines.h:33:49: note: expanded from macro 'GAD_DEPRECATED_ATTRIBUTE'

    #define GAD_DEPRECATED_ATTRIBUTE __attribute__((deprecated))

                                                    ^

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.m:106:19: warning: 'gender'

    is deprecated [-Wdeprecated-declarations]

              request.gender = genderValue;

                      ^

    In module 'GoogleMobileAds' imported from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:

    /Users/hinetsmartsolutions/whole1/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/

    GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADReques

    t.h:92:47: note: 'gender' has been explicitly marked deprecated here

    @property(nonatomic, assign) GADGender gender GAD_DEPRECATED_ATTRIBUTE;

                                                  ^

    In module 'GoogleMobileAds' imported from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:

    /Users/hinetsmartsolutions/whole1/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/

    GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMob

    ileAdsDefines.h:33:49: note: expanded from macro 'GAD_DEPRECATED_ATTRIBUTE'

    #define GAD_DEPRECATED_ATTRIBUTE __attribute__((deprecated))

                                                    ^

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.m:116:14: warning:

    'tagForChildDirectedTreatment:' is deprecated:  use

    [GADMobileAds.sharedInstance.requestConfiguration

    tagForChildDirectedTreatment] [-Wdeprecated-declarations]

        [request tagForChildDirectedTreatment:childDirected.boolValue];

                 ^

    In module 'GoogleMobileAds' imported from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:

    /Users/hinetsmartsolutions/whole1/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/

    GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADReques

    t.h:126:5: note: 'tagForChildDirectedTreatment:' has been explicitly marked

    deprecated here

        GAD_DEPRECATED_MSG_ATTRIBUTE(

        ^

    In module 'GoogleMobileAds' imported from

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTRequestFactory.h:5:

    /Users/hinetsmartsolutions/whole1/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/

    GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMob

    ileAdsDefines.h:25:56: note: expanded from macro

    'GAD_DEPRECATED_MSG_ATTRIBUTE'

    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))

                                                           ^

    4 warnings generated.

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTMobileAd.m:39:17: warning:

    'configureWithApplicationID:' is deprecated: Use

    -[GADMobileAds.sharedInstance startWithCompletionHandler:]

    [-Wdeprecated-declarations]

      [GADMobileAds configureWithApplicationID:appId];

                    ^

    In module 'GoogleMobileAds' imported from

    /Users/hinetsmartsolutions/whole1/ios/Pods/Headers/Public/Firebase/Firebase.

    h:161:

    /Users/hinetsmartsolutions/whole1/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/

    GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GADMobile

    Ads.h:83:5: note: 'configureWithApplicationID:' has been explicitly marked

    deprecated here

        GAD_DEPRECATED_MSG_ATTRIBUTE("Use -[GADMobileAds.sharedInstance

        startWithCompletionHandler:]");

        ^

    In module 'GoogleMobileAds' imported from

    /Users/hinetsmartsolutions/whole1/ios/Pods/Headers/Public/Firebase/Firebase.

    h:161:

    /Users/hinetsmartsolutions/whole1/ios/Pods/Google-Mobile-Ads-SDK/Frameworks/

    GoogleMobileAdsFramework-Current/GoogleMobileAds.framework/Headers/GoogleMob

    ileAdsDefines.h:25:56: note: expanded from macro

    'GAD_DEPRECATED_MSG_ATTRIBUTE'

    #define GAD_DEPRECATED_MSG_ATTRIBUTE(s) __attribute__((deprecated(s)))

                                                           ^

    /Users/hinetsmartsolutions/flutter/.pub-cache/hosted/pub.dartlang.org/fireba

    se_admob-0.9.3+2/ios/Classes/FLTMobileAd.m:97:79: warning: implicit

    conversion loses integer precision: 'FLTMobileAdStatus' to 'int'

    [-Wshorten-64-to-32]

      NSString *statusString = (NSString *)statusToString[[NSNumber

      numberWithInt:_status]];

                                                          ~

                                                          ^~~~~~~

    2 warnings generated.

    While building module 'uni_links' imported from

    /Users/hinetsmartsolutions/whole1/ios/Runner/GeneratedPluginRegistrant.m:146

    :

    In file included from <module-includes>:1:

    In file included from

    /Users/hinetsmartsolutions/whole1/build/ios/Release-iphoneos/uni_links/uni_l

    inks.framework/Headers/uni_links-umbrella.h:13:

    /Users/hinetsmartsolutions/whole1/build/ios/Release-iphoneos/uni_links/uni_l

    inks.framework/Headers/UniLinksPlugin.h:4:4: warning: pointer is missing a

    nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified)

    [-Wnullability-completeness]

    + (instancetype)sharedInstance;

       ^

    /Users/hinetsmartsolutions/whole1/build/ios/Release-iphoneos/uni_links/uni_l

    inks.framework/Headers/UniLinksPlugin.h:4:4: note: insert '_Nullable' if the

    pointer may be null

    + (instancetype)sharedInstance;

       ^

                    _Nullable 

    /Users/hinetsmartsolutions/whole1/build/ios/Release-iphoneos/uni_links/uni_l

    inks.framework/Headers/UniLinksPlugin.h:4:4: note: insert '_Nonnull' if the

    pointer should never be null

    + (instancetype)sharedInstance;

       ^

                    _Nonnull 

    /Users/hinetsmartsolutions/whole1/build/ios/Release-iphoneos/uni_links/uni_l

    inks.framework/Headers/UniLinksPlugin.h:7:33: warning: block pointer is

    missing a nullability type specifier (_Nonnull, _Nullable, or

    _Null_unspecified) [-Wnullability-completeness]

          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;

                                    ^

    /Users/hinetsmartsolutions/whole1/build/ios/Release-iphoneos/uni_links/uni_l

    inks.framework/Headers/UniLinksPlugin.h:7:33: note: insert '_Nullable' if

    the block pointer may be null

          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;

                                    ^

                                     _Nullable

    /Users/hinetsmartsolutions/whole1/build/ios/Release-iphoneos/uni_links/uni_l

    inks.framework/Headers/UniLinksPlugin.h:7:33: note: insert '_Nonnull' if the

    block pointer should never be null

          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;

                                    ^

                                     _Nonnull

    2 warnings generated.

    2 warnings generated.

    While building module 'uni_links' imported from

    /Users/hinetsmartsolutions/whole1/ios/Runner/GeneratedPluginRegistrant.m:146

    :

    In file included from <module-includes>:1:

    In file included from

    /Users/hinetsmartsolutions/whole1/build/ios/Release-iphoneos/uni_links/uni_l

    inks.framework/Headers/uni_links-umbrella.h:13:

    /Users/hinetsmartsolutions/whole1/build/ios/Release-iphoneos/uni_links/uni_l

    inks.framework/Headers/UniLinksPlugin.h:4:4: warning: pointer is missing a

    nullability type specifier (_Nonnull, _Nullable, or _Null_unspecified)

    [-Wnullability-completeness]

    + (instancetype)sharedInstance;

       ^

    /Users/hinetsmartsolutions/whole1/build/ios/Release-iphoneos/uni_links/uni_l

    inks.framework/Headers/UniLinksPlugin.h:4:4: note: insert '_Nullable' if the

    pointer may be null

    + (instancetype)sharedInstance;

       ^

                    _Nullable 

    /Users/hinetsmartsolutions/whole1/build/ios/Release-iphoneos/uni_links/uni_l

    inks.framework/Headers/UniLinksPlugin.h:4:4: note: insert '_Nonnull' if the

    pointer should never be null

    + (instancetype)sharedInstance;

       ^

                    _Nonnull 

    /Users/hinetsmartsolutions/whole1/build/ios/Release-iphoneos/uni_links/uni_l

    inks.framework/Headers/UniLinksPlugin.h:7:33: warning: block pointer is

    missing a nullability type specifier (_Nonnull, _Nullable, or

    _Null_unspecified) [-Wnullability-completeness]

          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;

                                    ^

    /Users/hinetsmartsolutions/whole1/build/ios/Release-iphoneos/uni_links/uni_l

    inks.framework/Headers/UniLinksPlugin.h:7:33: note: insert '_Nullable' if

    the block pointer may be null

          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;

                                    ^

                                     _Nullable

    /Users/hinetsmartsolutions/whole1/build/ios/Release-iphoneos/uni_links/uni_l

    inks.framework/Headers/UniLinksPlugin.h:7:33: note: insert '_Nonnull' if the

    block pointer should never be null

          restorationHandler:(void (^)(NSArray *_Nullable))restorationHandler;

                                    ^

                                     _Nonnull

    2 warnings generated.

    2 warnings generated.

    /Users/hinetsmartsolutions/Library/Developer/Xcode/DerivedData/Runner-brolrq

    ntkohexeagyddincnwgplq/Build/Intermediates.noindex/Runner.build/Release-ipho

    neos/Runner.build/Script-A7C5966809FD8F2E7C79C278.sh: line 2:

    /Users/hinetsmartsolutions/whole1/ios/Pods/Target Support

    Files/Pods-Runner/Pods-Runner-frameworks.sh: Permission denied

    Command PhaseScriptExecution failed with a nonzero exit code

    note: Using new build system

    note: Building targets in parallel

    note: Planning build

    note: Using build description from disk



Encountered error while building for device.

""





**_Step 2 (require): submit proof of purchasing the license on http://verify.inspireui.com_**





**_Important Note:_**
- We will close the ticket if missing the proof of purchase on step 2. The screenshot purchase attaches here will be invalid.
- Kindly create only *One Ticket* & includes all the issues, that would help us focus to resolve it better. 
- If your all ticket was resolved & closed, but want to create a new ticket, just do simple step by linking to the previous verified ID (you don't need to submit the form again), for example, #Ticket-ID

Thank you so much for your time 😊

**Comment by minhcasi** [comment]
@yasernet1989 - once change the bundle ID and app name, pls try to change from the info.plist file by following this guide https://docs.inspireui.com/fluxstore/delivery/ - instead of changing from the XCode UI - thank you!

**Comment by yasernet1989** [comment]
@minhcasi sorry, your suggestion didn't work for me. Instead, i found this solution here which worked for me. https://developer.apple.com/forums/thread/123790?answerId=386841022#386841022
I'm sharing for anyone's may face same issue,

Thanks

**Comment by minhcasi** [comment]
@yasernet1989 - thank you so much for your sharing, i will update document regarding this issue 👍
